Table validator: Fix bootlint errors

EricDunsworth opened the pull request:
This commit fixes a few bootlint errors that were introduced in #8699.

The purpose of the change in #8699 that caused the bootlint errors seemed to be to visually-bold certain checkbox and radio button labels. This commit reverts that change, which consequently removes the bolding.

@thekodester @duboisp @GormFrank FYI

EricDunsworth commented:
@duboisp Btw I don't mind removing the ``<b>`` elements if you'd prefer to retain the pre-#8699 behaviour (i.e. no bolding at all on checkbox/radio button labels).

GormFrank commented on src/other/tablevalidator/tablevalidator-en.hbs:
```diff
@@ -26,35 +26,25 @@
-			</label>
-		</div>
+		<label class="checkbox-inline" for="chkHassum">
+			<input type="checkbox" value="chkHassum" id="chkHassum" /> <b>Has a summary group</b>
```
Remove `<b>` everywhere

GormFrank reviewed (changes requested):
Pre-approved during technical review as long as `<b>` elements are removed

EricDunsworth committed:
Table validator: Fix bootlint errors

This commit fixes a few bootlint errors that were introduced in #8699.

The purpose of the change in #8699 that caused the bootlint errors seemed to be to visually-bold certain checkbox and radio button labels. This commit reverts that change, which consequently removes the bolding.
